handlers/profile/workspaces.py

```diff
@@ -16,11 +18,17 @@ def workspace_create(**params):
     if request.method == "POST":
         title = request.form.get("title")
         slug = request.form.get("slug")
+        user = params["user"]
+        user_id = user.get_id
 
         if title and slug:
-            Workspace.create(title=title, slug=slug)
+            workspace = Workspace.create(title=title, slug=slug)
```

Comment:
Zdej pa pejd še v create() metodo v Workspace modelu in daj objekt v return (ker zdaj vrne samo besedo "Success", ne pa objekta).

Reply:
Tukaj ne vem kater objekt moram returnati oziroma kje. Čim dam return workspace je spodnja koda unreachable.

Reply:
@JanVerderber Funkcije lahko vrnejo nekaj, ali pa ne.

Funkcija, ki ne vrne ničesar:

```python
def some_function():
    some_num = 5

some_function()  # poženi funkcijo
result = some_function()  # poženi funkcijo in dobi ven rezultat
print(result)  # izpiše None, ker funkcije ne vrne ničesar
```

Zdaj pa poglejmo še funkcijo, ki pa vrne neko vrednost:

```python
def some_function():
    some_num = 5
    return some_num

result = some_function()  # poženi funkcijo in dobi ven rezultat
print(result)  # izpiše: 5
```

Funkcija lahko vrne tudi več vrednosti:

```python
def some_function():
    some_num = 5
    some_str = "hello"
    return some_num, some_str

result = some_function()  # poženi funkcijo in dobi ven rezultat
print(result)  # izpiše: (5, 'hello')

print(result[0])  # izpiše 5
print(result[1])  # izpiše hello
```

Kot vidiš, je result v zgornjem primeru tuple z dvema vrednostima: (5, 'hello'). Namesto da nazaj dobimo tuple, lahko result razdelimo v dve spremenljivki:

```python
def some_function():
    some_num = 5
    some_str = "hello"
    return some_num, some_str

result_num, result_str = some_function()  # poženi funkcijo in dobi ven rezultat

print(result_num)  # izpiše 5
print(result_str)  # izpiše hello
```

Je zdaj kaj bolj jasno? :) Metoda Workspace.create() mora vrniti nek rezultat, najbolje da kar tri stvari: `return True, workspace, "Success"`. Potem pa v handlerju te tri stvari prestrežeš z `success, workspace, message`.

Poglej si, kako je narejeno v User modelu.